install_opi5.sh

```diff
@@ -4,55 +4,89 @@ if id "$1" >/dev/null 2>&1; then
     echo 'user found'
 else
     echo "creating pi user"
-    useradd pi -b /home
+    useradd pi -m -b /home -s /bin/bash
     usermod -a -G sudo pi
-    mkdir /home/pi
-    chown -R pi /home/pi
 fi
 echo "pi:raspberry" | chpasswd
 
-apt-get update
+apt-get update --quiet
+
+before=$(df --output=used / | tail -n1)
+# clean up stuff
+echo 'Purging snaps'
+# get rid of snaps
+rm -rf /var/lib/snapd/seed/snaps/*
+rm -f /var/lib/snapd/seed/seed.yaml
+apt-get purge --yes --quiet lxd-installer lxd-agent-loader
+apt-get purge --yes --quiet snapd
+
+# remove bluetooth daemon
+apt-get purge --yes --quiet bluez
+
+apt-get --yes --quiet autoremove
+
+after=$(df --output=used / | tail -n1)
+freed=$(( before - after ))
+
+echo "Freed up $freed KiB"
+
+# run Photonvision install script
 wget https://git.io/JJrEP -O install.sh
 chmod +x install.sh
 
-sed -i 's/# AllowedCPUs=4-7/AllowedCPUs=4-7/g' install.sh
+sed -i 's/# AllowedCPUs=4-7/AllowedCPUs=0-7/g' install.sh
 
 ./install.sh -n -q
```

Comment:
Since you disabled networking support / NetworkManager via the `-n` in your previous commit, PhotonVision can no longer set the device hostname.

Reply:
This is temporary, and part of the reason this is still draft. I have some more commits that I'll push to improve networking on OPis.

Reply:
The latest pushes have re-enabled network management.